src/api/index.js

```diff
@@ -23,26 +23,30 @@ export function getHosts() {
   return doGet('/api/inventory/v1/hosts');
 }
 
-export function downloadPlaybook(selectedIds) {
-  return new Promise((resolve) => {
-    const tab = downloadPlaybooksParamCreator({
-      selectedRemediations: selectedIds,
-      options: { baseURL: API_BASE },
-    }).then((res) => {
-      const { search, pathname } = res.urlObj;
-      const url = `${API_BASE}${pathname}${search}`;
-      window.location.assign(url);
-      return true;
-    });
-
-    const handle = setInterval(async () => {
-      if (tab) {
-        clearInterval(handle);
-        resolve();
-      }
-    }, 500);
-  });
-}
+export const downloadPlaybook = async (selectedIds) => {
+  try {
+    if (selectedIds.length > 1) {
+      downloadPlaybooksParamCreator({
+        selectedRemediations: selectedIds,
+        options: { baseURL: API_BASE },
+      }).then((res) => {
+        const { search, pathname } = res.urlObj;
+        const url = `${API_BASE}${pathname}${search}`;
+        window.location.assign(url);
+        return true;
+      });
+    } else {
+      const res = await remediationsApi.downloadPlaybooks({
```

Comment:
Should this not be `remediationsApi.getRemediationPlaybook`?

Reply:
Also... you are calling the `remediationsApi.downloadPlaybooks` function as I think you describe is not working above? ... hm. fishier and fishier. :D